src/pull_automerge/core.clj

```diff
@@ -103,13 +110,14 @@
   (println "Merge result:" (result :status))
   (if (result :error)
     (remove-label options org repo pull-number label
-      (str "MERGE ERROR: " (result :error)))))
+      (str "MERGE ERROR: " (result :error)))
+    (delete-branch options org repo branch)))
```

Comment:
I may be reading this incorrectly, but if there is an error merging and it removes the automerge tag, will it also delete the branch regardless?

Reply:
https://clojuredocs.org/clojure.core/if